### DISCUSSION TITLE
WebUI shows only blank screen.

### DISCUSSION THREAD

**Discussion by rspierenburg** [music-assistant/support#3234] [Q&A] [answered]
So I've installed the Integration along with the Server Add-on.  The server seems to be running fine but whenever I click on either the WebUI button or the Music Assistant sidebar button I'm redirected to a blank page.

Home Assistant Core 2024.11.3
Supervisor 2024.11.4

Music Assistant Server 2.3.3
Music Assistant Integration 2024.11.5

Here is the log for the Server Add-on:

[32m2024-11-27 08:27:00.527 INFO (MainThread) [music_assistant.music] Sync task for Music Assistant completed[0m
[33m2024-11-27 08:32:42.199 WARNING (MainThread) [aiorun] Stopping the loop[0m
[32m2024-11-27 08:32:42.200 INFO (MainThread) [aiorun] Entering shutdown phase.[0m
[32m2024-11-27 08:32:42.200 INFO (MainThread) [aiorun] Executing provided shutdown_callback.[0m
[32m2024-11-27 08:32:42.200 INFO (MainThread) [root] shutdown requested![0m
[32m2024-11-27 08:32:42.200 INFO (MainThread) [music_assistant] Stop called, cleaning up...[0m
[32m2024-11-27 08:32:42.217 INFO (MainThread) [aiorun] Waiting for executor shutdown.[0m
[32m2024-11-27 08:32:42.222 INFO (MainThread) [aiorun] Shutting down async generators[0m
[32m2024-11-27 08:32:42.223 INFO (MainThread) [aiorun] Closing the loop.[0m
[32m2024-11-27 08:32:42.224 INFO (MainThread) [aiorun] Leaving. Bye![0m
[32m2024-11-27 08:34:07.306 INFO (MainThread) [music_assistant] Starting Music Assistant Server (e74f27c2ad104be887e0a776f1605f99) version 2.3.3 - HA add-on: True - Safe mode: False[0m
[32m2024-11-27 08:34:07.312 INFO (MainThread) [music_assistant.cache] Initializing cache controller...[0m
[32m2024-11-27 08:34:07.638 INFO (MainThread) [music_assistant.music] Using a sync interval of 180 minutes.[0m
[32m2024-11-27 08:34:07.753 INFO (MainThread) [music_assistant.streams] Detected ffmpeg version 6.1.1 with libsoxr support[0m
[32m2024-11-27 08:34:07.753 INFO (MainThread) [music_assistant.streams] Starting server on  0.0.0.0:8097 - base url: http://192.168.1.2:8097[0m
[32m2024-11-27 08:34:07.760 INFO (MainThread) [music_assistant.webserver] Starting server on  172.30.32.1:8095 - base url: http://172.30.32.1:8095[0m
[32m2024-11-27 08:34:08.255 INFO (MainThread) [music_assistant] Loaded metadata provider fanart.tv[0m
[32m2024-11-27 08:34:08.256 INFO (MainThread) [music_assistant] Loaded metadata provider The Audio DB[0m
[32m2024-11-27 08:34:08.256 INFO (MainThread) [music_assistant] Loaded music provider Music Assistant[0m
[32m2024-11-27 08:34:08.256 INFO (MainThread) [music_assistant] Loaded metadata provider MusicBrainz[0m
[32m2024-11-27 08:34:08.257 INFO (MainThread) [music_assistant] Loaded player provider Playergroup[0m
[32m2024-11-27 08:34:08.327 INFO (MainThread) [music_assistant.music] Sync task for Music Assistant completed[0m
[33m2024-11-27 08:35:58.313 WARNING (MainThread) [aiorun] Stopping the loop[0m
[32m2024-11-27 08:35:58.313 INFO (MainThread) [aiorun] Entering shutdown phase.[0m
[32m2024-11-27 08:35:58.313 INFO (MainThread) [aiorun] Executing provided shutdown_callback.[0m
[32m2024-11-27 08:35:58.313 INFO (MainThread) [root] shutdown requested![0m
[32m2024-11-27 08:35:58.313 INFO (MainThread) [music_assistant] Stop called, cleaning up...[0m
[32m2024-11-27 08:35:58.337 INFO (MainThread) [aiorun] Waiting for executor shutdown.[0m
[32m2024-11-27 08:35:58.343 INFO (MainThread) [aiorun] Shutting down async generators[0m
[32m2024-11-27 08:35:58.344 INFO (MainThread) [aiorun] Closing the loop.[0m
[32m2024-11-27 08:35:58.345 INFO (MainThread) [aiorun] Leaving. Bye![0m
[32m2024-11-27 08:35:59.961 INFO (MainThread) [music_assistant] Starting Music Assistant Server (e74f27c2ad104be887e0a776f1605f99) version 2.3.3 - HA add-on: True - Safe mode: False[0m
[32m2024-11-27 08:35:59.968 INFO (MainThread) [music_assistant.cache] Initializing cache controller...[0m
[32m2024-11-27 08:35:59.997 INFO (MainThread) [music_assistant.music] Using a sync interval of 180 minutes.[0m
[32m2024-11-27 08:36:00.064 INFO (MainThread) [music_assistant.streams] Detected ffmpeg version 6.1.1 with libsoxr support[0m
[32m2024-11-27 08:36:00.065 INFO (MainThread) [music_assistant.streams] Starting server on  0.0.0.0:8097 - base url: http://192.168.1.2:8097[0m
[32m2024-11-27 08:36:00.072 INFO (MainThread) [music_assistant.webserver] Starting server on  172.30.32.1:8095 - base url: http://172.30.32.1:8095[0m
[32m2024-11-27 08:36:00.571 INFO (MainThread) [music_assistant] Loaded metadata provider fanart.tv[0m
[32m2024-11-27 08:36:00.571 INFO (MainThread) [music_assistant] Loaded metadata provider The Audio DB[0m
[32m2024-11-27 08:36:00.572 INFO (MainThread) [music_assistant] Loaded music provider Music Assistant[0m
[32m2024-11-27 08:36:00.573 INFO (MainThread) [music_assistant] Loaded metadata provider MusicBrainz[0m
[32m2024-11-27 08:36:00.573 INFO (MainThread) [music_assistant] Loaded player provider Playergroup[0m
[32m2024-11-27 08:36:00.632 INFO (MainThread) [music_assistant.music] Sync task for Music Assistant completed[0m
[33m2024-11-27 08:39:34.979 WARNING (MainThread) [aiorun] Stopping the loop[0m
[32m2024-11-27 08:39:34.980 INFO (MainThread) [aiorun] Entering shutdown phase.[0m
[32m2024-11-27 08:39:34.980 INFO (MainThread) [aiorun] Executing provided shutdown_callback.[0m
[32m2024-11-27 08:39:34.980 INFO (MainThread) [root] shutdown requested![0m
[32m2024-11-27 08:39:34.981 INFO (MainThread) [music_assistant] Stop called, cleaning up...[0m
[32m2024-11-27 08:39:34.989 INFO (MainThread) [aiorun] Waiting for executor shutdown.[0m
[32m2024-11-27 08:39:34.992 INFO (MainThread) [aiorun] Shutting down async generators[0m
[32m2024-11-27 08:39:34.992 INFO (MainThread) [aiorun] Closing the loop.[0m
[32m2024-11-27 08:39:34.993 INFO (MainThread) [aiorun] Leaving. Bye![0m
[32m2024-11-27 08:39:36.660 INFO (MainThread) [music_assistant] Starting Music Assistant Server (e74f27c2ad104be887e0a776f1605f99) version 2.3.3 - HA add-on: True - Safe mode: True[0m
[32m2024-11-27 08:39:36.667 INFO (MainThread) [music_assistant.cache] Initializing cache controller...[0m
[32m2024-11-27 08:39:36.695 INFO (MainThread) [music_assistant.music] Using a sync interval of 180 minutes.[0m
[32m2024-11-27 08:39:36.763 INFO (MainThread) [music_assistant.streams] Detected ffmpeg version 6.1.1 with libsoxr support[0m
[32m2024-11-27 08:39:36.763 INFO (MainThread) [music_assistant.streams] Starting server on  0.0.0.0:8097 - base url: http://192.168.1.2:8097[0m
[32m2024-11-27 08:39:36.770 INFO (MainThread) [music_assistant.webserver] Starting server on  172.30.32.1:8095 - base url: http://172.30.32.1:8095[0m
[33m2024-11-27 08:39:49.990 WARNING (MainThread) [aiorun] Stopping the loop[0m
[32m2024-11-27 08:39:49.991 INFO (MainThread) [aiorun] Entering shutdown phase.[0m
[32m2024-11-27 08:39:49.991 INFO (MainThread) [aiorun] Executing provided shutdown_callback.[0m
[32m2024-11-27 08:39:49.991 INFO (MainThread) [root] shutdown requested![0m
[32m2024-11-27 08:39:49.991 INFO (MainThread) [music_assistant] Stop called, cleaning up...[0m
[32m2024-11-27 08:39:49.999 INFO (MainThread) [aiorun] Waiting for executor shutdown.[0m
[32m2024-11-27 08:39:50.001 INFO (MainThread) [aiorun] Shutting down async generators[0m
[32m2024-11-27 08:39:50.002 INFO (MainThread) [aiorun] Closing the loop.[0m
[32m2024-11-27 08:39:50.003 INFO (MainThread) [aiorun] Leaving. Bye![0m
[32m2024-11-27 08:39:51.664 INFO (MainThread) [music_assistant] Starting Music Assistant Server (e74f27c2ad104be887e0a776f1605f99) version 2.3.3 - HA add-on: True - Safe mode: False[0m
[32m2024-11-27 08:39:51.671 INFO (MainThread) [music_assistant.cache] Initializing cache controller...[0m
[32m2024-11-27 08:39:51.697 INFO (MainThread) [music_assistant.music] Using a sync interval of 180 minutes.[0m
[32m2024-11-27 08:39:51.765 INFO (MainThread) [music_assistant.streams] Detected ffmpeg version 6.1.1 with libsoxr support[0m
[32m2024-11-27 08:39:51.765 INFO (MainThread) [music_assistant.streams] Starting server on  0.0.0.0:8097 - base url: http://192.168.1.2:8097[0m
[32m2024-11-27 08:39:51.773 INFO (MainThread) [music_assistant.webserver] Starting server on  172.30.32.1:8095 - base url: http://172.30.32.1:8095[0m
[32m2024-11-27 08:39:52.292 INFO (MainThread) [music_assistant] Loaded metadata provider fanart.tv[0m
[32m2024-11-27 08:39:52.293 INFO (MainThread) [music_assistant] Loaded metadata provider The Audio DB[0m
[32m2024-11-27 08:39:52.293 INFO (MainThread) [music_assistant] Loaded music provider Music Assistant[0m
[32m2024-11-27 08:39:52.293 INFO (MainThread) [music_assistant] Loaded metadata provider MusicBrainz[0m
[32m2024-11-27 08:39:52.294 INFO (MainThread) [music_assistant] Loaded player provider Playergroup[0m
[32m2024-11-27 08:39:52.354 INFO (MainThread) [music_assistant.music] Sync task for Music Assistant completed[0m
[33m2024-11-27 08:39:54.098 WARNING (MainThread) [aiorun] Stopping the loop[0m
[32m2024-11-27 08:39:54.098 INFO (MainThread) [aiorun] Entering shutdown phase.[0m
[32m2024-11-27 08:39:54.098 INFO (MainThread) [aiorun] Executing provided shutdown_callback.[0m
[32m2024-11-27 08:39:54.098 INFO (MainThread) [root] shutdown requested![0m
[32m2024-11-27 08:39:54.098 INFO (MainThread) [music_assistant] Stop called, cleaning up...[0m
[32m2024-11-27 08:39:54.102 INFO (MainThread) [aiorun] Waiting for executor shutdown.[0m
[32m2024-11-27 08:39:54.103 INFO (MainThread) [aiorun] Shutting down async generators[0m
[32m2024-11-27 08:39:54.103 INFO (MainThread) [aiorun] Closing the loop.[0m
[32m2024-11-27 08:39:54.104 INFO (MainThread) [aiorun] Leaving. Bye![0m
[32m2024-11-27 08:40:00.601 INFO (MainThread) [music_assistant] Starting Music Assistant Server (e74f27c2ad104be887e0a776f1605f99) version 2.3.3 - HA add-on: True - Safe mode: False[0m
[32m2024-11-27 08:40:00.609 INFO (MainThread) [music_assistant.cache] Initializing cache controller...[0m
[32m2024-11-27 08:40:00.634 INFO (MainThread) [music_assistant.music] Using a sync interval of 180 minutes.[0m
[32m2024-11-27 08:40:00.701 INFO (MainThread) [music_assistant.streams] Detected ffmpeg version 6.1.1 with libsoxr support[0m
[32m2024-11-27 08:40:00.701 INFO (MainThread) [music_assistant.streams] Starting server on  0.0.0.0:8097 - base url: http://192.168.1.2:8097[0m
[32m2024-11-27 08:40:00.708 INFO (MainThread) [music_assistant.webserver] Starting server on  172.30.32.1:8095 - base url: http://172.30.32.1:8095[0m
[32m2024-11-27 08:40:01.220 INFO (MainThread) [music_assistant] Loaded metadata provider fanart.tv[0m
[32m2024-11-27 08:40:01.220 INFO (MainThread) [music_assistant] Loaded metadata provider The Audio DB[0m
[32m2024-11-27 08:40:01.220 INFO (MainThread) [music_assistant] Loaded music provider Music Assistant[0m
[32m2024-11-27 08:40:01.221 INFO (MainThread) [music_assistant] Loaded metadata provider MusicBrainz[0m
[32m2024-11-27 08:40:01.222 INFO (MainThread) [music_assistant] Loaded player provider Playergroup[0m
[32m2024-11-27 08:40:01.301 INFO (MainThread) [music_assistant.music] Sync task for Music Assistant completed[0m

**Comment by OzGav** [comment]
The log suggests the server isn’t running fine as it keeps shutting down… but it looks like something has requested it to be shut down? There is a toggle in the configuration for the add on to start in safe mode. Try that and see if it will stay up long enough to get into the UI

**Reply by rspierenburg** [reply]
That was me, trying to restart the server to see if that would help.

**Reply by OzGav** [reply]
Try a different browser

**Reply by rspierenburg** [reply]
Same deal on both Edge and Firefox

**Reply by OzGav** [reply]
Firefox is a known problem at times. Edge should work but you can also try chrome. Beyond that look at firewall or other network settings that may be interfering. Do you have other add ons that work ok?

**Reply by rspierenburg** [reply]
Yep, all my other addons work (Docker Wyze Bridge, Frigate, Zigbee2MQTT).
Chrome was the first browser I tried.

**Reply by OzGav** [reply]
There is likely something in your network or other setup as no one else is reporting this.

**Reply by rspierenburg** [reply, marked as answer]
Thanks for all the quick replies, yeah I figured it must be network related somehow.  Just wasn't sure if there might have been something in the logs that I had missed.
